docs/procedures/general_checks.md

```diff
@@ -0,0 +1,110 @@
+      We usually go for $\Delta Q_x = -0.01$ and $\Delta Q_y = 0.012$.
+      Don't drive on the tune, you don't want to see what happens then ;)
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Have the Losses Applications Open</summary>
+      <p> When kicking from Multiturn, keep an eye on the losses application for unexpected spikes.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Make Sure to Log All Info</summary>
+      <p> Starting in Run 3 a decent amount of automatic logging should be implemented.
+      However, one should always check and make sure to log all relevant information: fill number, location of measurement files etc.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Analyse as the Data Comes</summary>
+      <p> As much as possible, let's make sure people analysing the kick data do not fall behind the kickers.
+      If you're a kicker, coordinate with you analyst to stay in sync.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Give Sensible Names to Complete Sets of Data</summary>
+      <p> When an analysis is done on a complete group of kicks, try to find a descriptive name.
+      See the [info][../info.md] page for naming conventions.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Log Anything of Interest in the Logbook</summary>
+      <p> It doesn't hurt to have a lot of information.
+      </p></details>
+
+## General Corrections Caveats
+
+- [ ] <details class="nodeco"><summary>Beware of the Corrections Signs</summary>
+      - [ ] <details class="nodeco"><summary>Calculated Global Corrections</summary>
+            <p> The calculated global correction are really corrections and should be trimmed in with a positive sign.
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Skew Magnets</summary>
+            <p> All skew magnets are inversed in LSA / MAD-X conventions and the calculated corrections for these should be trimmed in with an opposite sign (that means calculated correction * -1).
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Triplets</summary>
+            <p> The triplets are tricky as all three are powered in series with a common knob, then some have an additional trim.
+            Calculated corrections should be *sign-swapped for Q1 and Q3 but not for Q2*.
+            See with the EIC if you have any doubts.
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Local Corrections</summary>
+            <p> Decide on a convention for how we output the corrections (omc3 or bbsrc) and how to use them? Do checks on friday for that!
+            </p></details>
+
+- [ ] <details class="nodeco"><summary>Knob Creation Permission</summary>
+      <p> It is not enough to be logged in as `lhcop` to create knobs for corrections.
+      Both Tobias and Ewen have the permissions, if they aren't present see with the EIC.
+      </p></details>
```

Comment:
They might not have the permission but they can give themself or anyone in the team the permission

Reply:
Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)